operator/option/config.go

```diff
@@ -515,6 +523,7 @@ func (c *OperatorConfig) Populate() {
 
 	c.AWSReleaseExcessIPs = viper.GetBool(AWSReleaseExcessIPs)
 	c.AWSEnablePrefixDelegation = viper.GetBool(AWSEnablePrefixDelegation)
+	c.AWSUsePrimaryAddress = viper.GetBool(AWSUsePrimaryAddress)
```

Comment:
Is this flag still necessary now that you populate the CRD field via CNI conf?

Reply:
So, I could not find a clean way to propagate the flag from the CRD spec in places like 

https://github.com/cilium/cilium/blob/d23d3d366d72124537d8d0ff3c8df5acd494f854/pkg/aws/ec2/ec2.go#L327-L330

Do you have ideas on how this can be done ? This is tricky because `GetInstances()` is listing interfaces directly and does not access the node object.

Reply:
Ah, I see. Yeah, in that case a global flag is probably the better option for the operator, since we can't really set the `usePrimary` flag per node (as we do with the other CNI-populated CRD fields).